src/Controller/TranscriptionsController.php

```diff
@@ -127,10 +136,20 @@ public function save($sentenceId, $script) {
             );
         }
 
-        $this->setViewVars(array_filter(array($saved)), $sentenceId);
-        $this->render('view');
         if (!$saved) {
-            return $this->response->withStatus(400, 'Bad transcription');
+            return $this->response->withStatus(400, __('The transcription could not be saved.'));
```

Comment:
:smile: Well, it is certainly not common practice to use the status line as an error message displayed to users, but nothing’s really preventing you from doing it. This message is typically not translated and only read by developers (API users). I assume that this is temporary and that you are aware that the Transcription model is already providing detailed error messages.